launch

```diff
@@ -21,7 +21,7 @@ while getopts 'b' option; do
 done
 
 if [ "$build_image" == "1" ]; then
-    docker build . --tag irregularengineering/slackbot:latest
+    docker build . --tag irregularengineering/slackbot:latest --no-cache
```

Comment:
was having buggy behaviour, can be removed if desired...

Reply:
maybe also add `--force-rm`?  The idea is to force a build if you're using this option, vs using the cache / already downloaded images.

Also paste your error here if you get it again.